app/mailers/devise_notify_mailer.rb

```diff
@@ -1,11 +1,15 @@
 class DeviseNotifyMailer < Devise::Mailer
   require "notifications/client"
 
+  ALLOWLIST = %w[madetech.com levellingup.gov.uk communities.gov.uk].freeze
```

Comment:
Does defining this list in the code introduce any potential security risks?

Reply:
It does give away which domains we allow, but I don't think that sharing this information increases the attack surface.